geode-core/src/main/java/org/apache/geode/internal/cache/TXCommitMessage.java

```diff
@@ -416,8 +416,10 @@ void send(TXLockId lockId) {
                 setRecipientsSendData(Collections.singleton(indivRecip.next()), processor, rcl);
               }
             } else {
-              Set<InternalDistributedMember> tempNotificationOnlyMembers = new HashSet();
-              Set<InternalDistributedMember> tempTransactionMembers = new HashSet();
+              Set<InternalDistributedMember> tempNotificationOnlyMembers =
+                  new HashSet<InternalDistributedMember>();
+              Set<InternalDistributedMember> tempTransactionMembers =
+                  new HashSet<InternalDistributedMember>();
```

Comment:
These should just be `new HashSet<>();` with the angle brackets present but empty. It's not necessary to specify the type as part of the constructor, as it's implicit from the `Set<InternalDistributedMember>` declaration, but it's still necessary to use empty angle brackets to show that the `HashSet` is parameterized. This comment also applies to the other changes in this commit.

Reply:
I hope that now is OK.

Reply:
Yes, thank you, and sorry for the confusion.